metaflow/plugins/cards/card_modules/chevron/renderer.py

```diff
@@ -0,0 +1,427 @@
+# -*- coding: utf-8 -*-
+
+import io
+from os import linesep, path
+
+try:
+    from collections.abc import Sequence, Iterator, Callable
+except ImportError:  # python 2
+    from collections import Sequence, Iterator, Callable
+try:
+    from .tokenizer import tokenize
+except (ValueError, SystemError):  # python 2
+    from tokenizer import tokenize
+
+
+import sys
+
+if sys.version_info[0] == 3:
+    python3 = True
+    unicode_type = str
+    string_type = str
+
+    def unicode(x, y):
+        return x
+
+
+else:  # python 2
+    python3 = False
+    unicode_type = unicode
+    string_type = basestring  # noqa: F821 (This is defined in python2)
+
+
+#
+# Helper functions
+#
+
+
+def _html_escape(string):
+    """HTML escape all of these " & < >"""
+
+    html_codes = {
+        '"': "&quot;",
+        "<": "&lt;",
+        ">": "&gt;",
+    }
+
+    # & must be handled first
+    string = string.replace("&", "&amp;")
+    for char in html_codes:
+        string = string.replace(char, html_codes[char])
```

Comment:
there must be some library to do this properly, to avoid script injection hacks etc?